lib/ansible/module_utils/network/common/utils.py

```diff
@@ -35,20 +35,13 @@
 import json
 
 from itertools import chain
-from socket import inet_aton
-from json import dumps
 
 from ansible.module_utils._text import to_text, to_bytes
 from ansible.module_utils.common._collections_compat import Mapping
 from ansible.module_utils.six import iteritems, string_types
 from ansible.module_utils import basic
 from ansible.module_utils.parsing.convert_bool import boolean
 
-# Backwards compatibility for 3rd party modules
```

Comment:
So, I think for this, we need to check which modules are importing these, and have them import directly.  IMO, there shouldn't be a need for modules to link here. Also, this helps prepare the for the move to ansible.netcommon